From d3a2e50f242e487b81d927ec2a0cf4bb6c9aed2f Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Thu, 3 Apr 2025 04:37:47 -0700
Subject: [PATCH] remove -lm for all targets

---
 prog/makefile.static | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/prog/makefile.static b/prog/makefile.static
index 820370e..61a9557 100644
--- a/prog/makefile.static
+++ b/prog/makefile.static
@@ -138,7 +138,7 @@ endif
 # Be sure LD_LIBRARY_PATH includes the appropriate library directories, such
 # as /usr/local/include, in which libwebp.so and/or libgif.so are installed
 # Use or modify an appropriate line below for ALL_LIBS:
-ALL_LIBS =	$(LEPTLIB) -ltiff -ljpeg -lpng -lz -lm
+ALL_LIBS =	$(LEPTLIB) -ltiff -ljpeg -lpng -lz
 #ALL_LIBS =	$(LEPTLIB) -ltiff -ljpeg -lpng -lwebp -lz -lm
 #ALL_LIBS =	$(LEPTLIB) -ltiff -ljpeg -lpng -lwebp -lwebpmux -lz -lm
 #ALL_LIBS =	$(LEPTLIB) -ltiff -ljpeg -lpng -lwebp -lopenjp2 -lz -lm
-- 
2.39.3 (Apple Git-146)