From 8413c741bb9515449ec6d8f69eafb821e1b3c624 Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Thu, 3 Apr 2025 04:18:32 -0700
Subject: [PATCH] remove m.lib dependency

---
 prog/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/prog/CMakeLists.txt b/prog/CMakeLists.txt
index 5f1057b..923959b 100644
--- a/prog/CMakeLists.txt
+++ b/prog/CMakeLists.txt
@@ -11,6 +11,9 @@ function(add_prog_target target)
         target_compile_definitions  (${target} PRIVATE -DLIBLEPT_IMPORTS)
     endif()
     target_link_libraries           (${target} leptonica)
+    if(WIN32)
+        set_property(TARGET ${target} PROPERTY LINK_OPTIONS "/NODEFAULTLIB:m.lib")
+    endif()
     set_target_properties           (${target} PROPERTIES FOLDER prog)
 endfunction(add_prog_target)
 ########################################
-- 
2.39.3 (Apple Git-146)
